frontend/src/lib/components/CommandBar/CommandBar.tsx

```diff
@@ -39,7 +39,9 @@ export function CommandBar(): JSX.Element | null {
     return (
         <CommandBarOverlay>
             <div
-                className="w-full h-160 max-w-lg bg-bg-3000 rounded overflow-hidden flex flex-col border shadow"
+                className={`w-full ${
```

Comment:
`shadow` amounts to a border-bottom in 3000 mode, which makes sense for buttons, but does not look great here IMO. Are we getting rid of the modal shadow in 3000 completely @corywatilo?